common/rdr/FileInStream.cxx

```diff
@@ -56,7 +56,7 @@ int FileInStream::pos()
   return ftell(file) + ptr - b;
 }
 
-int FileInStream::overrun(int itemSize, int nItems, bool wait)
+int FileInStream::overrun(int itemSize, int nItems, bool /* wait */)
```

Comment:
Yeah, I was on the fence about this. It's not necessary. However there are a bunch of no-op/stub methods in various header files which cause warnings to spill over into users of these headers.  If I want to include PixelBuffer.h (and many others), I have to add my own flags to hush warnings which I don't like. It's probably better to move some of these to the implementation files, would you agree?